homework31/hw31_task1.py

```diff
@@ -0,0 +1,47 @@
+# <HW31> Task1: Request to random web-site and print response details.
+
+import requests
+import random
+
+
+def random_url():
+    """Returns random URL from list"""
+    urls = [
+        "https://google.com",
+        "https://facebook.com",
+        "https://twitter.com",
+        "https://amazon.com",  # never responds for some reason
+        "https://apple.com"
+    ]
+    return random.choice(urls)
+
+
+def get_response_from(url: str):
```

Comment:
I think it is clear to name 
```suggestion
def get_response_from_url(url: str):
```

Reply:
True. It will work.